### ISSUE TITLE
Poperty Based Testing #Question

### ISSUE THREAD

**Issue by Polish-Civil** [toolisticon/axon-addons#19]
Hello, I'm not sure what are the main reasons of this repository.
It seems like the main focus is on the testing. I've mentioned it on the axon group but I would like to poke it again as i think it would be great addition to the axon way superior than example unit testing.

With that being said I'm also positive on contributing to such addon if it were being developed.

Food for thought links:

[jqwik - JUnit based property based testing framework](https://jqwik.net/)
[Youtube official AxonIQ video regarding it](https://www.youtube.com/watch?v=jlmuxJfzq-0)

Basically such addon would need to implement common generation for randomized domain types such as emails, ids, dates as well as predictable randomized sequences of commands using that data that would target specific aggregates within domain.

Thank you and good luck with current plan

**Comment by zambrovski** [comment]
Feel free to provide an example or an pull request of what you have in mind!
